linux/shim/service.go

```diff
@@ -211,6 +212,7 @@ func (s *Service) Resume(ctx context.Context, r *shimapi.ResumeRequest) (*google
 }
 
 func (s *Service) Exit(ctx context.Context, r *shimapi.ExitRequest) (*google_protobuf.Empty, error) {
+	unix.Unmount(filepath.Join(s.path, "rootfs"), 0)
```

Comment:
This should likely use a detached unmount which would allow the mount to go away even if it has leaked elsewhere... which is much more common than one might think.